app/uk/gov/hmrc/customs/inventorylinking/export/xml/MdgPayloadDecorator.scala

```diff
@@ -16,25 +16,28 @@
 
 package uk.gov.hmrc.customs.inventorylinking.export.xml
 
+import javax.inject.Singleton
+
 import org.joda.time.DateTime
 import org.joda.time.format.ISODateTimeFormat
-import uk.gov.hmrc.customs.inventorylinking.export.model.Ids
+import uk.gov.hmrc.customs.inventorylinking.export.model.actionbuilders.ValidatedPayloadRequest
 
 import scala.xml.NodeSeq
 
+@Singleton
 class MdgPayloadDecorator() {
```

Comment:
Do we still want references to MDG?

Reply:
This hits many files (> 20) so I suggest a follow up PR for this